Bootstrap/Podfile

```diff
@@ -1,3 +1,5 @@
+source 'https://github.com/CocoaPods/Specs.git'
```

Comment:
This should't be needed, you only need to define the `source 'https://github.com/CocoaPods/Specs.git'` if you have any private sources in the Podfile.